spec/system/candidate_interface/continuous_applications/course_selection/candidate_selecting_a_course_spec.rb

```diff
@@ -158,7 +158,13 @@ def when_the_course_is_full
   end
 
   def when_i_visit_the_course_choices_page
-    visit candidate_interface_continuous_applications_choices_path
+    @application_choice = current_candidate.current_application.application_choices.first
+    when_i_visit_my_applications
+  end
+
+  def then_i_should_see_that_the_course_is_full
```

Comment:
You've added a `should` here :grimacing:

Reply:
I know, I was like, "should" I continue to follow the "should" until we remove it or not? 😢